packages/sdk/src/exports.ts

```diff
@@ -44,15 +46,22 @@ export {
     DEFAULT_ENVIRONMENT_ID,
     EntryPointDiscovery
 } from './Config'
+export { validKeyTypeValues, KeyType, DEFAULT_KEY_TYPE, identityConfig } from './identity/identityConfig'
```

Comment:
Do we need to export `identityConfig` in the public API? As identities are typically accessed via explicit classes  (e.g. `EthereumKeyPairIdentity.generate()`), maybe this is not needed.

Reply:
It's needed by the CLI tool, so technically needs to be exported. But I'll mark it with `/** @internal */`